modules/KIWISchema.rnc

```diff
@@ -1783,8 +1783,20 @@ div {
         ## hybrid ISO is used as disk on e.g a USB Stick. By default
         ## the btrfs filesystem is used
         attribute hybridpersistent_filesystem {
-            "btrfs" | "fat"
+            "btrfs" | "fat" | "exfat"
         }
+    k.type.hybridpersistent_filename.attribute =
+        ## for the iso type only:
+        ## Set the filename to use for persistency file if ex/fat is used for
+        ## hybrid ISO write support. Default is "!Live OS's persistent storage.fs"
+        attribute hybridpersistent_filename { text }
+    k.type.hybridpersistent_filesize.attribute =
```

Comment:
both attributes will not find their way into the initrd profile environment, maybe this was just forgotten ?